client/blocks/jetpack-benefits/scan-history.tsx

```diff
@@ -0,0 +1,160 @@
+	const translate = useTranslate();
+	const siteScanState = useSelector( ( state ) => {
+		return getSiteScanState( state, siteId );
+	} );
+	const siteScanProgress = useSelector( ( state ) => {
+		return getSiteScanProgress( state, siteId );
+	} );
+	const requestingScanState = useSelector( ( state ) => {
+		return isRequestingJetpackScan( state, siteId );
+	} );
+	const requestingThreats = useSelector( ( state ) => {
+		return isRequestingJetpackScanThreatCounts( state, siteId );
+	} );
+	const threatCounts = useSelector( ( state ) => {
+		return state?.jetpackScan?.threatCounts?.data?.[ siteId ];
+	} );
+
+	// site scan state can be provisioning, scanning or idle. If missing from the state after request is ended, can assume an error
+	const scanState = siteScanState?.state;
+	const threatsFixedCount = threatCounts ? threatCounts.fixed : 0;
+
+	// scan is running now
+	if ( scanState === 'scanning' ) {
+		return (
+			<JetpackBenefitsCard
+				jestMarker="scanning"
+				headline={ translate( 'Site Scan' ) }
+				description={ <ProgressBar value={ siteScanProgress } total={ 100 } color="#069E08" /> }
+				stat={ translate( 'In Progress' ) }
+			/>
+		);
+	}
+
+	// scan getting ready to start
+	if ( scanState === 'provisioning' ) {
+		return (
+			<JetpackBenefitsCard
+				jestMarker="provisioning"
+				headline={ translate( 'Site Scan' ) }
+				description={ translate( 'Jetpack is preparing to scan your site.' ) }
+				stat="Preparing"
+			/>
+		);
+	}
+
+	// still requesting scan state
+	if ( requestingScanState ) {
+		return (
+			<JetpackBenefitsCard
+				jestMarker="loading"
+				headline={ translate( 'Site Scan' ) }
+				description={ translate( 'Waiting for scan status' ) }
+				stat={ translate( 'Loading' ) }
+				placeholder={ true }
+			/>
+		);
+	}
+
+	// something went wrong getting the scan state
+	if ( ! scanState ) {
+		return (
+			<JetpackBenefitsCard
+				jestMarker="error"
+				headline={ translate( 'Site Scan' ) }
+				description={ translate( 'Jetpack is having trouble scanning your site.' ) }
+			/>
+		);
+	}
+
+	const { threats, mostRecent } = siteScanState;
+	const mostRecentScanAgo = moment.utc( mostRecent.timestamp ).fromNow();
+
+	// show expended output for standalone scan products
+	if ( isStandalone ) {
+		return (
+			<JetpackBenefitsStandaloneCard
+				title={ translate( 'Site Scan' ) }
+				summary={ {
+					title: translate( 'Last Scan' ),
+					stat: mostRecentScanAgo,
+				} }
+				stats={ [
+					{
+						title: translate( 'Malware Threats Found' ),
+						stat: threats ? threats.length : '...',
+					},
+					{
+						title: translate( 'Threats Fixed (Lifetime)' ),
+						stat: requestingThreats ? '...' : threatsFixedCount,
+					},
+				] }
+			/>
+		);
+	}
+
+	let cardStat: TranslateResult = '';
+	let cardDescription: TranslateResult = '';
+	if ( threats.length > 0 ) {
+		cardStat = translate( '%(threatCount)d threat found', '%(threatCount)d threats found', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Database %(threatCount)d threat', 'Database %(threatCount)d threats' )` *ES Score: 8*